db/db.go

```diff
@@ -1,35 +1,81 @@
 package db
 
-import "fmt"
+import (
+	"errors"
+	"fmt"
+	"sync"
+)
 
 //import "strconv"
 
 var Datos = make(map[string]interface{})
```

Comment:
Please, define a general language (English preferred) in the whole project

Reply:
I already corrected and I am handling the structure that you recommended